src/store/modules/wallet.js

```diff
@@ -672,6 +681,34 @@ const actions = {
     return memo;
   },
 
+  async fetchSuggestedFollowList({ state, commit }) {
+    const currentFollowees = state.followees;
+    const currentFolloweesSet = new Set(currentFollowees);
+
+    // Cache check and return cached result if it's valid
+    if (
+      state.suggestedFollowList.length &&
+      !currentFollowees.some(user => state.suggestedFollowList.includes(user))
```

Comment:
Should we remove entry from `suggestedFollowList` if user click follow in suggested list?